Mathlib/Analysis/Convex/Between.lean

```diff
@@ -810,6 +810,18 @@ theorem wbtw_iff_sameRay_vsub {x y z : P} : Wbtw R x y z ↔ SameRay R (y -ᵥ x
     field_simp [(add_pos hr₁ hr₂).ne', hr₂.ne']
     ring
 
+theorem AffineIndependent.not_wbtw {a b c : P} (hT : AffineIndependent R ![a, b, c]) :
+    ¬ Wbtw R a b c := by
+  rw [affineIndependent_iff_not_collinear_set] at hT
+  exact mt (·.collinear) hT
+
+theorem AffineIndependent.not_wbtw_of_inj {ι} (i j k : ι)
+    (h : Function.Injective ![i, j, k]) (T : ι → P) (hT : AffineIndependent R T) :
```

Comment:
```suggestion
theorem AffineIndependent.not_wbtw_of_inj {ι} (i j k : ι)
    (h : Function.Injective ![i, j, k] {T : ι → P} (hT : AffineIndependent R T) :
```

Reply:
Done in 51f9673c381c57d22ac5aab666529fc3e2ea5e6f.